src/main/java/meditracker/MediTracker.java

```diff
@@ -17,15 +17,13 @@
  */
 public class MediTracker {
 
-    private Ui ui;
     private MedicationManager medicationManager;
     private DailyMedicationManager dailyMedicationManager;
 
     /**
      * Constructs a new MediTracker object and initializes the user interface.
```

Comment:
Perhaps, you could update this JavaDoc comment too as Ui is now static.
The rest of the JavaDoc comments were updated accordingly.

Reply:
Resolved in https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/73/commits/44c762e3450fa42e124449b4bd626bc60faceff5